lib/config.js

```diff
@@ -119,6 +120,7 @@ function Config() {
             this.data_dir = '~/Guerilla';
         }
         this.data_dir = expandTilde(this.data_dir);
+        this.temp_dir = temp.mkdirSync("guerillatemp");
     }
```

Comment:
what is the "directory" name if they supply a data_dir? If they don't supply it then you are thinking it would be Guerilla?